### PR TITLE
Final Release Prep

### PR DESCRIPTION
**Pull request by abejgonzalez**
See title.

TODO:
- Wait for #1442 
- Remove the README header on the tutorial/workshop
- Hammer release

**Related PRs / Issues**:

**Type of change**:
- [ ] Bug fix
- [ ] New feature
- [x] Other enhancement

**Impact**:
- [x] RTL change
- [ ] Software change (RISC-V software)
- [x] Build system change
- [ ] Other

**Contributor Checklist**:
- [x] Did you set `main` as the base branch?
- [x] Is this PR's title suitable for inclusion in the changelog and have you added a `changelog:<topic>` label?
- [x] Did you state the type-of-change/impact?
- [x] Did you delete any extraneous prints/debugging code?
- [x] Did you mark the PR with a `changelog:` label?
- [x] (If applicable) Did you update the conda `.conda-lock.yml` file if you updated the conda requirements file?
- [x] (If applicable) Did you add documentation for the feature?
- [x] (If applicable) Did you add a test demonstrating the PR?

- [x] (If applicable) Did you mark the PR as `Please Backport`?

### CODE REVIEW

**Review by jerryz123 — APPROVED** [review]
This LGTM. I think its fine to merge this in now, and handle the TODOs in follow-up PRs. Up to you.

**Comment by abejgonzalez** [comment]
Still waiting on FireSim PR: https://github.com/firesim/firesim/pull/1492 and the TODOs (including the Hammer PR). Then we can use this branch to circular bump everything. So let's hold off a bit.

File: tests/hello.c [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #include <stdio.h>
-#include "encoding.h"
+#include "riscv-pk/encoding.h"
```
Comment (jerryz123) [inline tests/hello.c:2]: I think if you change this to `<riscv-pk/encoding.h>`, you don't need to do the PREFIX_NO_MINUS, or even add the `-I` flag to CFLAGS.

**Review by jerryz123 — CHANGES_REQUESTED** [review]

File: tests/hello.c (hunk 1 above)
Comment (abejgonzalez) [reply tests/hello.c:2]: Done

**Comment by jerryz123** [comment]
@abejgonzalez I took the liberty of cleaning up this PR now that the uniquify fix is in  (thanks @joey0320).

No other chipyard PRs need to go in at this point, 1.9.1 is ready, pending firesim

**Review by jerryz123 — APPROVED** [review]

**Comment by jerryz123** [comment]
The run-cfg-finder job seems to have failed spuriously.
I'm going to push the release

**Pull request merged**